fastlane/Fastfile

```diff
@@ -37,11 +52,20 @@ SKIP_CODE_SIGNING_XCARGS = {
   CODE_SIGNING_REQUIRED: 'NO'
 }.freeze
 
-# Use this instead of getting values from ENV directly
-# It will throw an error if the requested value is missing
-def get_required_env(key)
-  UI.user_error!("Environment variable '#{key}' is not set. Have you setup #{USER_ENV_FILE_PATH} correctly?") unless ENV.key?(key)
-  ENV.fetch(key)
+before_all do
+  # Ensure we use the latest version of the Release Toolkit
+  check_for_toolkit_updates unless is_ci || ENV['FASTLANE_SKIP_TOOLKIT_UPDATE_CHECK']
+
+  # Check that the env files exist
+  unless is_ci || File.file?(USER_ENV_FILE_PATH)
+    error_message = <<~ERROR
+      #{USER_ENV_FILE_PATH} not found. Please copy env/user.env-example to #{USER_ENV_FILE_PATH} and fill in the values") using the following command:
+      \tcp #{PROJECT_ROOT_FOLDER}/fastlane/env/user.env-example #{USER_ENV_FILE_PATH}
+    ERROR
+    UI.user_error!(error_message)
```

Comment:
I am not sure `before_all` runs before any code that is defined in the global scope (at the top of the `Fastfile` outside any block)… and thus will run before the `Dotenv.load(…)` from line 26 which tries to load said file? 🤔

Reply:
Fixed in https://github.com/Automattic/simplenote-macos/commit/4359b9007bd86288d5077ca0aac4762f631320db